src/main/java/com/autotune/analyzer/services/ListStackLayers.java

```diff
@@ -34,53 +34,76 @@
 import static com.autotune.analyzer.utils.ServiceHelpers.addExperimentDetails;
 import static com.autotune.analyzer.utils.ServiceHelpers.addLayerDetails;
 
-public class ListAppLayers extends HttpServlet {
+public class ListStackLayers extends HttpServlet {
     /**
-     * Returns the list of applications monitored by autotune along with layers detected in the applications.
+     * Returns the list of applications stacks monitored by autotune along with layers detected in the applications.
      * <p>
      * Request:
-     * `GET /listAppLayers` returns the list of all applications monitored by autotune, and their layers.
+     * `GET /listStackLayers` returns the list of all application stacks monitored by autotune, and their layers.
      * <p>
-     * `GET /listAppLayers?experiment_name=<EXP_NAME>` returns the layers detected for the pods specific to the given experiment.
+     * `GET /listStackLayers?experiment_name=<EXP_NAME>` returns the layers detected for the pods specific to the given experiment.
      * <p>
      * Example JSON:
      * [
-     * {
-     * "experiment_name": "app1_autotune",
-     * “objective_function”: “transaction_response_time”,
-     * "slo_class": "response_time",
-     * “direction”: “minimize”
-     * "layers": [
-     * {
-     * "layer_level": 0,
-     * "layer_name": "container",
-     * "layer_details": "generic container tunables"
-     * },
-     * {
-     * "layer_level": 1,
-     * "layer_name": "openj9",
-     * "layer_details": "java openj9 tunables"
-     * }
-     * ]
-     * },
-     * {
-     * "experiment_name": "app2_autotune",
-     * “objective_function”: “performedChecks_total”,
-     * "slo_class": "throughput",
-     * “direction”: “maximize”
-     * "layers": [
-     * {
-     * "layer_level": 0,
-     * "layer_name": "container",
-     * "layer_details": "generic container tunables"
-     * },
-     * {
-     * "layer_level": 1,
-     * "layer_name": "hotspot",
-     * "layer_details": "java hotspot tunables"
-     * }
-     * ]
-     * }
+     *     {
+     *         "experiment_name": "autotune-max-http-throughput",
+     *         "experiment_id": "94f76772f43339f860e0d5aad8bebc1abf50f461712d4c5d14ea7aada280e8f3",
+     *         "objective_function": "request_count",
+     *         "hpo_algo_impl": "optuna_tpe",
+     *         "stacks": [
+     *             {
+     *                 "layers": [{
+     *                     "layer_id": "af07fd998199bf2d57f95dc18f2cc2311b72f6de11e7e949b566fcdc5ecb443b",
+     *                     "layer_level": 0,
+     *                     "layer_name": "container",
+     *                     "layer_details": "generic container tunables"
+     *                 }],
+     *                 "stack_name": "dinogun/autotune_optuna:0.0.5"
```

Comment:
Should we change this to kruize instead of dinogun?

Reply:
This is just an example listing of the API, it doesnt really matter.